### PR TITLE
PUBDEV-5081: Reduce urls that get sent to google analytics [WIP]

### PR DESCRIPTION
**Pull request by tomkraljevic**

### CODE REVIEW

File: h2o-core/src/main/java/water/util/GAUtils.java [hunk 1]
```diff
@@ -13,16 +13,40 @@
...
 public class GAUtils {
   public static void logRequest(String uri, Properties header) {
+    boolean logged = logRequest2(uri, header);
+
+    final boolean debug = false;
+    if (debug && logged) {
+      System.out.println("GA LOGGED:  " + uri);
+    }
+    else {
+      System.out.println("GA DROPPED: " + uri);
```
Comment (michalkurka) [inline h2o-core/src/main/java/water/util/GAUtils.java]: Isn't this going to pollute the output?

Maybe we can use logger and enable it for the trace level?

Comment (tomkraljevic) [reply h2o-core/src/main/java/water/util/GAUtils.java]: No, debug is final false.
Once we're happy with it we can delete that block entirely.

Comment (michalkurka) [reply h2o-core/src/main/java/water/util/GAUtils.java]: Because debug is "false" it will always print "GA DROPPED" regardless if it was dropped or not

Comment (tomkraljevic) [reply h2o-core/src/main/java/water/util/GAUtils.java]: sorry, you're right, i fixed it.

**Pull request merged**